### PR TITLE
minor ui changes

### PR DESCRIPTION
**Pull request by aramikm**
# Description
- removed the button and after every change it should try to decode
- Some more UI changes

# UI changes
Screenshot 2024-08-14 at 10 26 43 AM

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -160,15 +187,22 @@
...
 		margin-bottom: 1rem;
 		box-sizing: border-box;
 	}
+
+	table, th, td {
+		border: 1px solid black;
+		border-collapse: collapse;
+		padding-left: 5px;
+		padding-right: 5px;
+	}
```
Comment (wilwade) [inline src/routes/+page.svelte]: Two small adjustments suggested to expand the padding some:
image


```suggestion
	table {
		width: 100%;
	}

	table, th, td {
		border: 1px solid black;
		border-collapse: collapse;
		padding: 5px;
	}
```

**Review by wilwade — CHANGES_REQUESTED** [review]
One style suggestion and a need to run `npm run format` (No CI enforcement at this time for anything)

**Review by wesbiggs — APPROVED** [review]
Works for me

**Review by wilwade — APPROVED** [review]
🚢

**Pull request merged**